crates/oxide/src/scanner/allowed_paths.rs

```diff
@@ -0,0 +1,59 @@
+use ignore::{DirEntry, WalkBuilder};
+use lazy_static::lazy_static;
+use std::path::Path;
+
+lazy_static! {
```

Comment:
fyi now that we're on Rust 1.80 I think `lazy_static` can be replaced with `std::sync::LazyLock`